packages/terra-notification-dialog/src/NotificationDialog.scss

```diff
@@ -0,0 +1,126 @@
+:local {
+  .notification-dialog {
+    background: var(--notification-dialog-background, #fff);
+    border-radius: var(--notification-dialog-border-radius, 5px);
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+
+    @media screen and (max-width: 768px) {
+      width: calc(100% - 2 * var(--notification-dialog-small-horizontal-inset, 10px));
+    }
+  }
+
+  .notification-dialog-body {
+    background-color: var(--notification-dialog-body-color, #fff);
```

Comment:
Update variable from `--notification-dialog-body-color` to `--notification-dialog-body-background-color`